### DISCUSSION TITLE
Zebra Does Not Load Any Tweak Page/Description

### DISCUSSION THREAD

**Discussion by Jacobahalls** [wstyres/Zebra#2070] [Q&A]
![IMG_7627](https://github.com/zbrateam/Zebra/assets/170248/98df15c5-817f-4417-9a4f-729faa3f28da)
When selecting tweaks, themes, etc it never loads the screenshots, change logs, or description. I like Zebra a lot due to their search being far superior to Sileo.

**Comment by iAdam1n** [comment]
This is a known bug that we are looking into.

**Reply by Rafter9875** [reply]
У меня так

**Reply by latggg** [reply]
I have encountered this problem, too. Is there a solution? Unloading, reloading and restoring settings have all been tried, which is useless.

**Comment by latggg** [comment]
I have encountered this problem, too. Is there a solution? Unloading, reloading and restoring settings have all been tried, which is useless.> This is a known bug that we are looking into.

**Reply by iAdam1n** [reply]
No solution.
